app/Http/Controllers/Admin/CallchampionsController.php

```diff
@@ -2,7 +2,7 @@
 namespace App\Http\Controllers\Admin;
 use App\Http\Controllers\Controller;
 use App\Models\CallChampion;
-use App\Models\Users;
```

Comment:
How did this code work before? Was User not being used at all in the whole code?